src/dhanshri/Assignment_15/BankNestedSwitch.java

```diff
@@ -0,0 +1,116 @@
+package dhanshri.Assignment_15;
+
+public class BankNestedSwitch {
+	int iBalance = 5000, hBalance = 8000, bBalance = 10000, sBalance = 15000;
+	int iDebit=0, hDebit=0, bDebit=0, sDebit=0, iCredit=0, hCredit=0, bCredit=0, sCredit=0;
+
+	void operations(String bank, String op, int amt) {
+		switch (op) {
+		case "Debit":
+			switch (bank) {
+			case "ICICI":
+				if (iBalance >= amt) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.

Code snippet

int debit(int totalAmt, int debitAmt) {
		if (totalAmt < debitAmt) {
			return 0;
		} else
			return totalAmt - debitAmt;
}

totalAmt will your iciciAmt (instance variable)